Makefile

```
@@ -0,0 +1,56 @@
+# Copyright (c) 2021 Red Hat, Inc.
+# Copyright Contributors to the Open Cluster Management project
+
+BINDIR ?= output
+
+include build/Configfile
+
+USE_VENDORIZED_BUILD_HARNESS ?=
+
+ifndef USE_VENDORIZED_BUILD_HARNESS
+-include $(shell curl -s -H 'Authorization: token ${GITHUB_TOKEN}' -H 'Accept: application/vnd.github.v4.raw' -L https://api.github.com/repos/open-cluster-management/build-harness-extensions/contents/templates/Makefile.build-harness-bootstrap -o .build-harness-bootstrap; echo .build-harness-bootstrap)
+else
+-include vbh/.build-harness-vendorized
+endif
+
+default::
+	@echo "Build Harness Bootstrapped"
+
+.PHONY: deps
+deps:
+	 go get github.com/golangci/golangci-lint/cmd/golangci-lint@v1.24.0
+	 go mod tidy
+
+.PHONY: insights-client
+insights-client:
+	 CGO_ENABLED=0 go build -a -v -i -installsuffix cgo -ldflags '-s -w' -o $(BINDIR)/insights-client ./
+
+.PHONY: build
+build: insights-client
+
+.PHONY: build-linux
+build-linux:
+	make insights-client GOOS=linux
+
+.PHONY: lint
+lint:
+	# Flag GOGC=75 needed to avoid out of memory issue.
+	 GOGC=100 golangci-lint run --timeout=2m
```

`GOGC=100` is the default. It means garbage collection runs after heap size doubles.  I had to add this in the aggregator because it's a larger project, but most likely you don't need it here.
```suggestion
	 golangci-lint run --timeout=2m
```

---

I'm committing this minor change.